src/licensedcode/data/licenses/dpl.yml

```diff
@@ -0,0 +1,11 @@
+key: dpl
```

Comment:
Since this is versioned as 1.1, please rename (here and also the filenames) as `dpl-1.1`

Reply:
There exists already a license with this key https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/dpl-1.1.yml

Reply:
name it with this key then  `defensive-patent-1.1`